pom.xml

```diff
@@ -164,18 +164,4 @@
             <url>https://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
-
-    <profiles>
-      <profile>
-        <id>jdk11</id>
```

Comment:
I remember trying to remove this at one time. But something didn't work. Has that been fixed somewhere since last I tried?

Reply:
> But something didn't work. Has that been fixed somewhere since last I tried?

@rsandell I am not overwhelmed by your specificity.

See the change to `Constructor.java`. The Java 11 compiler does not accept `<tt>`, but it does accept `{@code}`.